cypress/e2e/court-cases-index/sortingCases.cy.ts

```diff
@@ -115,7 +113,7 @@ describe("Sorting cases", () => {
     checkCasesOrder([1, 3, 2, 0])
 
     // Sort descending by court name
-    cy.get("#court-name-sort").click()
+    cy.get("#court-name-sort").find(".upArrow").click()
```

Comment:
I would add extra tests for the arrow and the `#court-name-sort` element. They test two different behaviours. Does that make sense?

And I test the `.downArrow` as well.